src/index.html

```diff
@@ -19,9 +19,367 @@
+        </button>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="imac image"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__name-price">Price:</p>
+          <p class="card__num-price">$2,199</p>
+        </div>
+        <button
+          class="card__button"
+          type="button"
+        >
+          Buy
+        </button>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="imac image"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__name-price">Price:</p>
+          <p class="card__num-price">$2,199</p>
+        </div>
+        <button
+          class="card__button"
+          type="button"
+        >
+          Buy
+        </button>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="imac image"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__name-price">Price:</p>
+          <p class="card__num-price">$2,199</p>
+        </div>
+        <button
+          class="card__button"
+          type="button"
+        >
+          Buy
+        </button>
```

Comment:
The product card content is repeated multiple times with the same data. Consider using a template engine or JavaScript to dynamically generate these cards to avoid redundancy and improve maintainability.